### PR TITLE
Feature/link item review

### PR DESCRIPTION
**Pull request by Jguerra47**
- Added relation between Item and Review model
- Added new migration file to handle a new column with its foreign key
- Updated factory to retrieve a foreign key from items table during reviews seeding

image

NOTE: New dependency added **_doctrine/dbal_**: Let handle operations on tables without losing the current data.

### CODE REVIEW

**Review by DanielHernandezO — DISMISSED** [review]

**Comment by DanielHernandezO** [comment]
First of all we have to fix some bugs, after that you need to merge Develop branch into this branch to be approved, this happens because we want to clean up the history of PRs, therefore this PR will be blocked until the bugs are fixed

**Review by Juanda0 — APPROVED** [review]

**Pull request merged**
